internal/olm/manager.go

```diff
@@ -28,14 +28,16 @@ import (
 const (
 	DefaultVersion = "latest"
 	DefaultTimeout = time.Minute * 2
+	// DefaultNamespace is the namespace where OLM is installed
+	DefaultNamespace = "olm"
 )
 
 type Manager struct {
-	Client  *Client
-	Version string
-	Timeout time.Duration
-
-	once sync.Once
+	Client    *Client
+	Version   string
+	Timeout   time.Duration
+	Namespace string
```

Comment:
Same here. It could be the watch namespace of the operator or the namespace where the operator is deployed in. So, we need to clarifies. 
```suggestion
	OlmNamespace string
```